lib/recomb_map.c

```diff
@@ -210,7 +210,9 @@ recomb_map_sample_poisson(recomb_map_t *self, gsl_rng *rng, double start)
     double left_bound, mass_to_next_recomb;
 
     left_bound = self->discrete ? start + 1 : start;
-    mass_to_next_recomb = gsl_ran_exponential(rng, 1.0);
+    do {
+        mass_to_next_recomb = gsl_ran_exponential(rng, 1.0);
+    } while (mass_to_next_recomb == 0);
```

Comment:
I can't find out whether 0 is guaranteed to be the same as 0.0 - can you change to 0.0 for peace of mind please? (Or educate me!)

Reply:
I also can't find out through a quick search, which is scary, so I'll just change it 😅